features/0023-did-exchange/README.md

```diff
@@ -97,152 +97,14 @@ No errors are sent in timeout situations. If the inviter or invitee wishes to re
-    "label": "Alice",
-    "recipientKeys": ["8HH5gYEeNc3z7PYXmd54d4x6qAfCNrqQqEB3nS7Zfu7K"],
-    "serviceEndpoint": "https://example.com/endpoint",
-    "routingKeys": ["8HH5gYEeNc3z7PYXmd54d4x6qAfCNrqQqEB3nS7Zfu7K"]
-}
-```
-
-Base 64 URL Encoded, with whitespace removed:
-
-```text
-eyJAdHlwZSI6ImRpZDpzb3Y6QnpDYnNOWWhNcmpIaXFaRFRVQVNIZztzcGVjL2Nvbm5lY3Rpb25zLzEuMC9pbnZpdGF0aW9uIiwiQGlkIjoiMTIzNDU2Nzg5MDA5ODc2NTQzMjEiLCJsYWJlbCI6IkFsaWNlIiwicmVjaXBpZW50S2V5cyI6WyI4SEg1Z1lFZU5jM3o3UFlYbWQ1NGQ0eDZxQWZDTnJxUXFFQjNuUzdaZnU3SyJdLCJzZXJ2aWNlRW5kcG9pbnQiOiJodHRwczovL2V4YW1wbGUuY29tL2VuZHBvaW50Iiwicm91dGluZ0tleXMiOlsiOEhINWdZRWVOYzN6N1BZWG1kNTRkNHg2cUFmQ05ycVFxRUIzblM3WmZ1N0siXX0=
-```
-Example URL:
-```text
-http://example.com/ssi?c_i=eyJAdHlwZSI6ImRpZDpzb3Y6QnpDYnNOWWhNcmpIaXFaRFRVQVNIZztzcGVjL2Nvbm5lY3Rpb25zLzEuMC9pbnZpdGF0aW9uIiwiQGlkIjoiMTIzNDU2Nzg5MDA5ODc2NTQzMjEiLCJsYWJlbCI6IkFsaWNlIiwicmVjaXBpZW50S2V5cyI6WyI4SEg1Z1lFZU5jM3o3UFlYbWQ1NGQ0eDZxQWZDTnJxUXFFQjNuUzdaZnU3SyJdLCJzZXJ2aWNlRW5kcG9pbnQiOiJodHRwczovL2V4YW1wbGUuY29tL2VuZHBvaW50Iiwicm91dGluZ0tleXMiOlsiOEhINWdZRWVOYzN6N1BZWG1kNTRkNHg2cUFmQ05ycVFxRUIzblM3WmZ1N0siXX0=
-```
-
-Invitation URLs can be transferred via any method that can send text, including an email, SMS, posting on a website, or via a QR Code.
-
-Example URL encoded as a QR Code:
-
-![exampleqr](exampleqr.png)
-
-
-#### Invitation Publishing
-
-The _inviter_ will then publish or transmit the invitation URL in a manner available to the intended _invitee_. After publishing, we have entered the _invited_ state.
-
-#### Invitation Processing
-
-When they _invitee_ receives the invitation URL, there are two possible user flows that depend on the SSI preparedness of the individual. If the individual is new to the SSI universe, they will likely load the URL in a browser. The resulting page will contain instructions on how to get started by installing software or a mobile app. That install flow will transfer the invitation message to the newly installed software.
-A user that already has those steps accomplished will have the URL received by software directly. That software will base64URL decode the string and can read the invitation message directly out of the `c_i` query parameter, without loading the URL.
-
-> **NOTE**: In receiving the invitation, the base64url decode implementation used **MUST**
-> correctly decode padded and unpadded base64URL encoded data.
-
-If the _invitee_ wants to accept the invitation, they will use the information present in the invitation message to prepare the request
+The DID Exchange Protocol is proceeded by either knowledge of a resolvable DID, or by a `oob-invitation/%VER/invitation` or `oob-request/%VER/request` message from the [Out Of Band Protocols RFC](../0434-outofband/README.md). The information from either the resolved DID Document or the `service` attribute of the `invitation` or `request` message is used to construct the `request` message to start the protocol.
```

Comment:
This paragraph needs to be fixed - remove the reference to the separate `invitation` and `request` OOB messages, now that the `request` one doesn't exist.

Reply:
+1